examples/chip-tool/README.md

```diff
@@ -101,18 +101,25 @@ with the target cluster name and the target command name
 ### How to parse a setup code
 
 To parse a setup code, run the built executable with the `payload` cluster name
-and the `parse` command
+and the `parse-qr-code` command
 
-    $ chip-tool payload parse code
+    $ chip-tool payload parse-qr-code code
 
 #### QR Code
 
-    $ chip-tool payload parse "CH:#####"
+    $ chip-tool payload parse-qr-code "CH:#####"
 
 #### QR Code with optional Vendor Info
 
-    $ chip-tool chip-tool payload parse "CH:#####"
+    $ chip-tool chip-tool payload parse-qr-code "CH:#####"
 
 #### Manual Setup Code
 
-    $ chip-tool payload parse :#####"
+    $ chip-tool payload parse-qr-code :#####"
+
+# Using the Client for Additional Data Payload
+
+To parse an additional data payload, run the built executable with the `payload`
+cluster name and the `parse-additional-data` command
+
+    $ chip-tool payload parse-additional-data "#####"
```

Comment:
- The term `additional-data` is very generic. How about giving it a specific name like `RotatingDeviceIdPayload` or move it under an appropriate namespace.
- Existing `SetupPayload` object supports optionalVendorData, can't we use that fields to represent this additional-data ?

Reply:
The additionaldata is actually generic intentionally, Currently it only include the rotating device id payload but it can include more member in the future.